chapter_philosophy_and_refactoring.asciidoc

```diff
@@ -277,6 +283,9 @@ selenium.common.exceptions.NoSuchElementException: Message: Unable to locate
 element: h1
 ----
 
+// DAVID: Could we take artistic license and break this line after NoSuchElementException:?
+// It would read more nicely.
```

Comment:
no, the tests are based on running with an 80-character terminal width, so i have to break things exactly that way, pretty much...  exceptions are possible!  but they would be equivalent to putting a hashtag-ignore on that line, ie making it untested...